test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
I would go ahead and delete everything under `test`. We will use Rspec for testing in the future.

Reply:
@lucasocon why was this marked as resolved?